### PR TITLE
Support for Elasticsearch and Graphite

### PR DESCRIPTION
**Pull request by jongear**
## v17.2.1.0 / 2017 Jul 13
* **Fix** - Bugfix for IniConfig.GetValue(string key, bool defaultValue) - previously, this method never actually pulled values from the config
* **Add** - `LogglyAppender` and `ElasticsearchLogAppender` as starter appenders for Logger
* **Add** - `DatadogReporter` and `GraphiteReporter` as starter appenders for Metrics
* **Update** - `AsyncEventReporter.PostEvent()` is now virtual and can be overriden by derived classes

```csharp
[GuaranteedRate.Sextant "17.2.1.0"]
```

### CODE REVIEW

**Comment by jongear** [comment]
@jeffpsherman @brianbegygr I have finished updating the Readme and believe this PR is ready for review

File: GuaranteedRate.Sextant/Metrics/IReporter.cs [hunk 1]
```diff
@@ -0,0 +1,10 @@
+﻿
+namespace GuaranteedRate.Sextant.Metrics
+{
+    public interface IReporter
+    {
+        void AddCounter(string name, long value);
+        void AddGauge(string name, long value);
+        void AddMeter(string name, long value);
```
Comment (jeffpsherman) [inline GuaranteedRate.Sextant/Metrics/IReporter.cs]: What about tags?

File: CHANGELOG.md [hunk 2]
```diff
@@ -155,19 +181,19 @@ in case of duplicates.
 >Added the ability to have Loggly load tags from config rather than be explicitly 
 >passed in.
 
-* **Update**  - Loggly.cs overloaded init method that doesn't do a force reload
+* **Update**  - Logger.cs overloaded init method that doesn't do a force reload
```
Comment (jeffpsherman) [inline CHANGELOG.md]: You renamed Loggly->Logger in the changelog.
Global search and replace error?

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,38 +1,64 @@
+## v17.2.1.0 / 2017 Jul 13
```
Comment (jeffpsherman) [inline CHANGELOG.md]: Can you please add some more context - we're going from Loggly and Datadog dependent to a implementation independent model with initial implementations for Loggly + ElasticCache, and Datadog + Graphana.

This seems to be a breaking change, what needs to change?  
Do users just need to update references to Logging and Metrics, or have configs changed as well?

File: CHANGELOG.md (hunk 2 above)
Comment (jongear) [inline CHANGELOG.md]: Technically Brian renamed it, I just leached off his hard work. But great eye 😄 

I will correct this

File: CHANGELOG.md (hunk 3 above)
Comment (jongear) [inline CHANGELOG.md]: 👍  thanks Jeff. I expanded the changelog docs.  Additionally, the readme has been expanded a great deal.

https://github.com/Guaranteed-Rate/GuaranteedRate.Sextant/tree/remove-loggly

File: GuaranteedRate.Sextant/Metrics/IReporter.cs (hunk 1 above)
Comment (jongear) [inline GuaranteedRate.Sextant/Metrics/IReporter.cs]: I'll talk to devops about this.  The tcp endpoint doesn't seem to support tags.  

Tags seem to only work under http posts http://graphite.readthedocs.io/en/latest/events.html

**Comment by jeffpsherman** [comment]
👍

**Pull request merged**